packages/synapse-react-client/src/components/IconList.tsx

```diff
@@ -25,7 +32,13 @@ const IconList: React.FunctionComponent<IconListProps> = props => {
         noMatch = true
         return
       } else {
-        return <IconSvg key={el} {...iconConfig} />
+        return (
+          <IconSvg
+            style={{ fontSize: iconFontSize }}
```

Comment:
I think this is fine. Wondering if it's possible to use `sx` instead of `style` but guessing that might not be easy because
* You might have to deeply merge props
* Not all of the IconSvg items use MUI's `SvgIcon`, which is the component that actually uses `sx`